lib/routes/podcast/boost.dart

```diff
@@ -51,40 +48,24 @@ class _BoostWidgetState extends State<BoostWidget> {
                 width: 88,
                 child: GestureDetector(
                   behavior: HitTestBehavior.translucent,
-                  onLongPress: () => showDialog(
+                  onTap: () => showDialog(
                     useRootNavigator: true,
```

Comment:
Since we have views that must hid the app content such as the pin code we should set the `useRootNavigator` to false.

Reply:
Interesting, to be honest, I didn't know what this flag was intended for, thanks for your comment and further research now I got it 🙏